x-pack/plugins/apm/server/routes/historical_data/has_historical_agent_data.ts

```diff
@@ -20,8 +26,10 @@ export async function hasHistoricalAgentData(apmEventClient: APMEventClient) {
       ],
     },
     body: {
+      terminate_after: 1,
```

Comment:
It looks like this property was used in the wrong place, should we remove the params variable and move its content to the search function on line 36? Or maybe, type the params so TS would warn us about that.

Reply:
`terminate_after`  is supported both as a top-level property, and inside the body. It's nice to have it co-located with `track_total_hits` and `size` but I don't have any strong opinions where it should go.